taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    def __str__(self):
+        return self.username
+
+    class Meta:
+        permissions = AbstractUser._meta.permissions
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(Driver, blank=True)

```

Comment:
You are using `Driver` directly in your `Car` model. However, the checklist recommends using `AUTH_USER_MODEL` instead. It's a good practice to use `settings.AUTH_USER_MODEL` to refer to the user model. This way, if you decide to change the user model in the future, you won't have to update it everywhere in your code.